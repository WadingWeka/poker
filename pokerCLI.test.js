const pokerapp = require("./pokerCLI");

describe("Check print hand output", () => {
    test('ordred card set', () => {
        let hand = [[1,1],[12,0],[10,3]];
        expect(pokerapp.getHandAsStr(hand)).toMatchSnapshot();
    });

    // test('ordred card set', () => {
    //     let hand = [[1,1],[12,0],[10,3]];
    //     expect(pokerapp.getHandAsStr(hand)).toMatchSnapshot();
    // });
});