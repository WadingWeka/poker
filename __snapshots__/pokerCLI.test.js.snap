// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check print hand output ordred card set 1`] = `
Array [
  "┌──────────────┐ ┌──────────────┐ ┌──────────────┐",
  "│ A♦           │ │ Q♣           │ │10♠           │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│              │ │              │ │              │",
  "│            A♦│ │            Q♣│ │           10♠│",
  "└──────────────┘ └──────────────┘ └──────────────┘",
]
`;
